### PR TITLE
Astro-591 - Adding Getting Started Content

### PR DESCRIPTION
**Pull request by im-cr**
### https://rocketcom.atlassian.net/browse/ASTRO-591

**Description:**
- Adding updated 'getting started' page
- Adding 'getting started for designers' page with content for Astro design resources
- Update to internal navigation include dropdown for getting started and sub-pages for designers

### CODE REVIEW

File: _content/_data/navigation.json [hunk 1]
```diff
@@ -5,7 +5,21 @@
 	},
 	{
 		"label": "Getting Started",
-		"url": "/getting-started"
+		"url": "/getting-started",
+		"items": [
+			{
+				"label" : "README",
+				"url" : "/getting-started/readme"
```
Comment (sknep) [inline _content/_data/navigation.json:12]: Do you know if this is the URL where Kat wanted the page posted at? It's kind of a weird url segment for the web, but if it's what she suggested, then nbd

File: _content/components/button.md [hunk 2]
```diff
@@ -4,10 +4,10 @@ path: /components/button
 date: Last Modified
 layout: components.template.njk
 title: Button
-demo: https://rocketcom.bitbucket.io/html-demos/button.html
+demo: https://astro-components.netlify.app/iframe.html?id=components-buttons--all-button-variants&viewMode=story
```
Comment (sknep) [inline _content/components/button.md:7]: this looks like stuff from another branch but i don't think it'll break anything

File: _content/getting-started/_readme.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+# Getting Started
+
+Astro is a design system that brings together UX guidelines, an [HTML/CSS](/components/readme/) framework, and web components. Astro is for both designers and developers.
+
+Astro is provided by the United States Space Force and Missile System Center (SMC) and is currently approved for designing Enterprise Ground Services (EGS) and Satellite Operations (SATOPS) web apps.
+
+Astro’s designs stem from continuous exploration and research. This user-centered design approach includes interviews and design validation with many SMC and commercial space systems operators. By incorporating operators and subject matter experts in our research and gaining their feedback during the design process, Astro ensures future space apps meet the needs of their users. Refer to the information provided on the [Research](/design-process/research/) page for more information on incorporating a user-centered approach in your application designs.
+
+Astro is a living design system, created and stewarded by [Rocket Communications Inc.](https://rocketcom.com/) in San Francisco. For questions, contact Rocket at: UXsupport@rocketcom.com.
```
Comment (sknep) [inline _content/getting-started/_readme.md:17]: Can we make that address a mailto link? I think its [address@domain.com](mailto:address@domain.com)

File: _content/getting-started/_readme.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+|       | Chrome | Firefox | Edge | Safari | Chrome (Android) | Safari (iOS) |
+|:-------|:--------:|:---------:|:------:|:--------:|:--------:|:--------:|
+| Astro |    &#x2715;    |    &#x2715;     | &#x2715;      |   &#x2715;     | unsupported | unsupported
+
+### Versioning
+
+Current version: Astro 4.5
+
+Last updated: Wed Jul 08 2020
```
Comment (sknep) [inline _content/getting-started/_readme.md:57]: Hmm, this is going to get out of date fast. @dmcalester I'm sure Kat was trying to be helpful by including it, but do we want to be on the hook for updating astrouxds.com with the latest release date and version?

also this isn't the components version, and it's not major.minor.patch, so i'm not even sure if it's real

File: _content/getting-started/designers.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+[Color](/design-guidelines/color/)
+[Typography](/design-guidelines/typography/)
+[Theme](/design-guidelines/theme/)
+[Grid](/design-guidelines/grid/)
+[Glossary](/design-guidelines/glossary/)
+[Compliance](/design-guidelines/compliance/)
+
+### Designer starter kit
+
+To download the Design starter kit, navigate to Astro Design Resources [Git Repository](https://github.com/RocketCommunicationsInc/astro-design-resources)
```
Comment (sknep) [inline _content/getting-started/designers.md:24]: I legit didn't even know we had this!

File: _content/getting-started/designers.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+### Sketch installation and use
+Download files from Astro Sketch Libraries:
+
+[Astro Dark Library](https://github.com/RocketCommunicationsInc/astro-design-resources/raw/master/Sketch/Astro%20Dark%20Library.sketch)
+[Astro Dark Sticker Sheet](https://github.com/RocketCommunicationsInc/astro-design-resources/raw/master/Sketch/Astro%20Dark%20Stickersheet.sketch)
+Astro Light Library (Coming soon…)
```
Comment (sknep) [inline _content/getting-started/designers.md:31]: These are coming soon before publish, or not? If not, lets cut

File: _content/getting-started/developers.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+In an effort to provide as close to native a development experience as possible, we’ve provided a set of [litElement-powered](https://lit-element.polymer-project.org/guide) [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) distributed in individual packages on [npm.org](https://www.npmjs.com/org/astrouxds). You can also see the full source code here and instructions for importing the components in a typical NodeJS project [here](https://github.com/RocketCommunicationsInc/astro-components/tree/feature/astro-tokens-static-assets#use-the-astro-web-components-in-a-nodejs-project).  
+  
+  
+### Astro CSS, Icons, and Fonts  
+We recognize that not all space application development projects are tooled for utilizing Web Components, so we also provide a CSS-only library containing name-spaced classes in addition to the icons and font files which are typically included within the components. You can [download today’s version](https://github.com/RocketCommunicationsInc/astro-components/tree/feature/astro-tokens-static-assets/src/components/rux-assets) (update link when live) of these assets, or install the package from [npm](link someday).
```
Comment (sknep) [inline _content/getting-started/developers.md:18]: Yeah these links aren't published yet, so we'll need to edit this on draft before it gets pushed to master

File: _content/getting-started/developers.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+### Getting Started  
+You can import Astro UXDS Web Components to an existing application using npm and litElement. Detailed instructions for using Astro components are available in the project’s README.  
+  
+If you’re considering a non-Web Component Astro implementation, directions for including Astro CSS and other static assets are available here. For those developing offline applications, be sure to review the special considerations for offline Astro assets.  
+  
+  
+### EGS Compliance  
+The Astro team has worked with EGS to establish the EGS Compliance requirements for developers targeting the EGS Platform. These requirements are specific to EGS development; these requirements are based off of common best practices and accessibility guidance and it is highly recommended for all users to incorporate them into their applications.  
+  
+EGS Design Compliance
```
Comment (sknep) [inline _content/getting-started/developers.md:46]: Do we need this line? it looks like a mistake

**Review by sknep — APPROVED** [review]
A few small tweaks and questions but nothing major. As long as we update these links when we release the next version of Astro components (with static assets package, since that totally changed the installation directions) we should be good

File: _content/getting-started/_readme.md (hunk 3 above)
Comment (im-cr) [reply _content/getting-started/_readme.md:17]: Yep good call out

**Pull request merged**